lightly/api/version_checking.py

```diff
@@ -33,23 +31,13 @@ def do_version_check(current_version: str):
     if current_process().name == 'MainProcess':
         with TimeoutDecorator(1):
             versioning_api = get_versioning_api()
-            current_version: str = versioning_api.get_latest_pip_version(
+            latest_version: str = versioning_api.get_latest_pip_version(
                 current_version=current_version)
-            latest_version: str = versioning_api.get_minimum_compatible_pip_version()
-
-            try:
-                if version_compare(current_version, latest_version) < 0:
-                    # local version is behind latest version
-                    pretty_print_latest_version(current_version, latest_version)
-            except ValueError:
-                pass
-                # error during version compare
-                # we just skip the version check
-
```

Comment:
What about this code block, why is it not necessary anymore?

Reply:
If we keep the block we will print a message whenever someone is not using the latest lightly version. This check has been broken for some time now (`current_version` was overridden by the latest version) so the message was never printed. I removed it now completely so that we're at least not printing the message on purpose.